resource-service/src/main/java/uk/gov/gchq/palisade/service/resource/service/ResourceServicePersistenceProxy.java

```diff
@@ -114,7 +116,8 @@ public ResourceServicePersistenceProxy(final PersistenceLayer persistence, final
                         )
                 ))
                 .map((AuditableResourceResponse resource) -> {
-                    LOGGER.debug("Got resource {} for request {}", resource, request);
+                    LOGGER.debug("Got resource {} for id {}", resource, request.getResourceId());
+                    LOGGER.trace(REQUEST_WAS, request);
```

Comment:
Still want trace messages in this file?

Reply:
Yeah, more logging means more noise. Some stuff isn't appropriate for debug, then some stuff is much more detailed than the previous logging statement, so gets bumped one down.